src/u2f/u2f_app.c

```diff
@@ -29,6 +29,96 @@ static const app_t _apps[] = {
                   "\x59\x43\x79\x4e\x88\x4f\x3d\x02\x3a\x82\x29\xfd",
         .name = "Google",
     },
+    {
```

Comment:
please add `sha256('...')` to each entry as above, so it's clear which website it is and how the app_id is constructed.

Reply:
in case you copied the hashes from some source, please share the source, so it's easier to double check the entries.

Reply:
Source is from here: https://github.com/trezor/trezor-firmware/blob/master/core/src/apps/webauthn/knownapps.py

Reply:
I'll also add that I'm clueless when it comes to this.

I'm sorry.

Reply:
+1 to sha256. @jengo9332 it seems those are auto-generated from https://github.com/trezor/trezor-firmware/tree/8cced84bc47f074b148abbaf595f2510c72ed6fb/common/defs/fido

See more here about App IDs: https://developers.yubico.com/U2F/App_ID.html

You can verify it manually like so:
$ echo -n 'https://www.dropbox.com/u2f-app-id.json' | sha256sum
c50f8a7b708e92f82e7a50e2bdc55d8fd91a22fe6b29c0cdf7805530842af581  -

Here's a couple more known IDs: https://github.com/github/SoftU2F/blob/master/SoftU2FTool/KnownFacets.swift

Reply:
No worries, thanks. knownapps.py seems to be generated from those sources:

https://github.com/trezor/trezor-firmware/tree/master/common/defs/fido

I'll check again tomorrow.

Reply:
I see, thanks.
I found an old list with known IDs and added them. However it's lacking Facebook and Fedora.

Reply:
I cleaned the file and added verified sha256('') to each entry. I also removed webauthn stuff since I just realized that webauthn != u2f.

However I'm still not sure why keeper has two different sha256sum, two different domains or?